### PR TITLE
fix: add Abhishek-00

### LINKED ISSUE #1670: [PROFILE ADDITION] Abhishek Kumar
### Profile screenshot

image

### PR DESCRIPTION
**Pull request by Abhishek-00**
resolves #1670 
## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/Abhishek-00.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/101923614/213878832-4282e481-ce0c-4a57-a3e6-6f3eeb688901.png",
+    "issueId": 1670,
+    "name": "Abhishek Kumar",
+    "username": "Abhishek-00"
+}
```
Comment (Panquesito7) [inline profiles/Abhishek-00.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Abhishek-00.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/101923614/213878832-4282e481-ce0c-4a57-a3e6-6f3eeb688901.png",
+    "issueId": 1670,
+    "name": "Abhishek Kumar",
+    "username": "Abhishek-00"
+}
+
```
Comment (Panquesito7) [inline profiles/Abhishek-00.json:7]:
```suggestion
```

File: profiles/Abhishek-00.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/101923614/213878832-4282e481-ce0c-4a57-a3e6-6f3eeb688901.png",
+    "issueId": 1670,
+    "name": "Abhishek Kumar",
+    "username": "Abhishek-00"
```
Comment (Panquesito7) [inline profiles/Abhishek-00.json:5]:
```suggestion
  "image": "https://user-images.githubusercontent.com/101923614/213878832-4282e481-ce0c-4a57-a3e6-6f3eeb688901.png",
  "issueId": 1670,
  "name": "Abhishek Kumar",
  "username": "Abhishek-00"
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Pradumnasaraf — COMMENTED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**